src/main/java/seedu/ui/Ui.java

```diff
@@ -117,4 +117,27 @@ public void printListOfMatchingIngredients(ArrayList<Ingredient> listOfIngredien
             printListOfIngredients(listOfIngredients, true);
         }
     }
+
+    /**
+     * Prints a reconfirm message and gets the reconfirm result.
+     * @return Boolean representing reconfirm status (y: confirm, n: abort)
+     */
+    public boolean getResetReconfirm() {
```

Comment:
Can change printing values to be kinda same style to have some structure like the other methods?
printLine()